<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceTest</class>
 <widget class="QDialog" name="InterfaceTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations du capteur</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxModele">
       <item>
        <property name="text">
         <string>FineOffset</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nexus</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCanal">
       <property name="text">
        <string>0</string>
       </property>
       <property name="placeholderText">
        <string>Canal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditidCourant">
       <property name="text">
        <string>15</string>
       </property>
       <property name="placeholderText">
        <string>IdCourant</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>108</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter Données</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ajouter Données Température</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditTemperature">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Température</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditHumidite">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Humidité</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Ajouter Données Vent</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>331</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEditVitesseMoyenne">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Vitesse Moyenne</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDirection">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Direction du vent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditVitesseMaximum">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Vitesse Maximum</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Ajouter Données Précipitations</string>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEditPrecipitation">
          <property name="text">
           <string>12</string>
          </property>
          <property name="placeholderText">
           <string>Précipitations</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
